eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -107,6 +107,7 @@ object TestConstants {
       paymentRequestExpiry = 1 hour,
       multiPartPaymentExpiry = 30 seconds,
       minFundingSatoshis = 1000 sat,
+      maxFundingSatoshis = Btc(1),
```

Comment:
Maybe also activate wumbo in Alice and bob's features if you put a wumbo value in here?
Or keep `16777216 sat` here to preserve existing tests, and only override it in wumbo-specific tests?